### PR TITLE
added info about test environment to contribute

### PR DESCRIPTION
**Pull request by szrtGreg**
## Description
Added Test environment section in CONTRIBUTING file.

## Where should the reviewer start?
Changes are only in one file - CONTRIBUTING.md

## Motivation and context
Related issue: #318 

## How has this been tested?
Manually -> check if file is correctly displyed

## Screenshots (if appropriate)
obraz

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ Login: `devadventcalendar@gmail.com`
 
 Password: `P@ssw0rd`
 
+## Test environment
+
+Test environment is running on dev.devadventcalendar.pl with automatic deployment from develop_uat branch.
```
Comment (WTobor) [inline CONTRIBUTING.md:85]: please make `dev.devadventcalendar.pl` a link

Comment (szrtGreg) [reply CONTRIBUTING.md:85]: changed to link

Comment (WTobor) [reply CONTRIBUTING.md:85]: cannot see it here :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ Password: `P@ssw0rd`
 
 ## Test environment
 
-Test environment is running on dev.devadventcalendar.pl with automatic deployment from develop_uat branch.
+Test environment is running on https://dev.devadventcalendar.pl/ with automatic deployment from develop_uat branch.
```
Comment (WTobor) [inline CONTRIBUTING.md:85]: in markdown you can create links
[link_name](link)
so please update file

Comment (szrtGreg) [reply CONTRIBUTING.md]: created link with correct structure

**Pull request merged**